pkgs/applications/misc/pigpio/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "pigpio";
+  version = "77";
+
+  src = fetchFromGitHub {
+    owner = "joan2937";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1pwi7gff3p3awv3v08d9lhycpmb1n44dbckv713rz162j4w290cd";
+  };
+
+  makeFlags = [ "prefix=$(out)" ];
+
+  postPatch = ''
+    substituteInPlace Makefile --replace '$(DESTDIR)/opt/pigpio/cgi' '$(DESTDIR)/$(prefix)/cgi'
+    substituteInPlace Makefile --replace 'ldconfig' '# ldconfig'
+  '';
+
+  meta = with stdenv.lib; {
```

```suggestion
  meta = with lib; {
```

stdenv.lib got recently deprecated.